api/client/options.proto

```diff
@@ -104,6 +105,7 @@ message H1ConnectionReuseStrategy {
 
 // TODO(oschaaf): Ultimately this will be a load test specification. The fact that it
 // can arrive via CLI is just a concrete detail. Change this to reflect that.
+// highest unused number is 38
```

Comment:
I don't think it's conventional to add in a helper comment like this, partially because it is likely to get out of sync with future changes.  I'd suggest removing it.

Reply:
This seems to be relatively standard practice. It provides the convenience of not having to look through the proto to find what the highest number is. I think it is unlikely to get out of sync because if you're editing the proto, the comment is readily available where the proto is, and it only needs to be updated when you're trying to add a new field to the proto.